cisco/asr1002-hx.yaml

```diff
@@ -0,0 +1,62 @@
+manufacturer: Cisco
+model: ASR1002-HX
+slug: asr1002-hx
+u_height: 2
+is_full_depth: false
+console_ports:
+  - name: con 0
+power_ports:
+  - name: P0
+  - name: P1
+interfaces:
+  - name: GigabitEthernet0/0/0
+    type: SFP (1GE)
```

Comment:
Interface types must reference the new (v2.7+) slugs; see the [schema validation](https://github.com/netbox-community/devicetype-library/blob/master/tests/schema.json) for all valid values.

(This should have caused the automated tests to fail, but we weren't catching `.yml` suffixes. This has been fixed.)

Reply:
Makes sense. Sorry for the PR in that setup. Went too quick.